** Mention Debian-specific changes in the Emacs NEWS file.
   Provided by: Rob Browning <rlb@defaultvalue.org>

---
 etc/NEWS |    5 +++++
 1 file changed, 5 insertions(+)

--- a/etc/NEWS
+++ b/etc/NEWS
@@ -21,6 +21,11 @@
 so we will look at it and add it to the manual.
 
 
+* Debian specific changes to Emacs
+
+Please see /usr/share/doc/emacs-snapshot-common/README.Debian.
+
+
 * About external Lisp packages
 
 
